.github/workflows/multi-cluster-on-isolated-networks.yaml

```diff
@@ -9,6 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: ./
+        id: test-cluster-1
```

Comment:
Is it just cosmetic? IIRC `id` in action is used to propagate the output to the downstream action. Not the case here?

Reply:
thx for review. True, I made it for two reasons:
 - I already [print output](https://github.com/AbsaOSS/k3d-action/blob/main/.github/workflows/multi-cluster-on-isolated-networks.yaml#L41)
 - I demonstrate output in documentation in this [file](https://github.com/AbsaOSS/k3d-action/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R165)